azurerm/resource_arm_postgresql_server.go

```diff
@@ -130,9 +130,8 @@ func resourceArmPostgreSQLServer() *schema.Resource {
 				ValidateFunc: validation.StringInSlice([]string{
 					string(postgresql.NineFullStopFive),
 					string(postgresql.NineFullStopSix),
+					string(postgresql.OneOne),
 					string(postgresql.OneZero),
-					string(postgresql.OneZeroFullStopZero),
```

Comment:
This version is still accepted by the API and we don't want to break users on this version so let's keep this version around